subprojects/packagefiles/soundtouch/meson.build

```diff
@@ -0,0 +1,108 @@
+project(
+    'soundtouch',
+    'c', 'cpp',
+    version: '2.3.2',
+    license: 'LGPL-2.1-or-later',
+    default_options: [
+        'debug=true',
+        'optimization=3',
+        'c_std=c11',
+        # to avoid Meson's warning under MSVC
+        'cpp_std=c++14',
```

Comment:
https://codeberg.org/soundtouch/soundtouch/commit/5e624fff737db6905ec049d002c583b030ce806d

Reply:
This wrap targets 2.3.2: https://codeberg.org/soundtouch/soundtouch/src/tag/2.3.2/CMakeLists.txt#L1-L4. Do you mean to add the flag for future proofing reasons?

Reply:
Yeah I saw that after. It's fine.